package.json

```diff
@@ -12,20 +12,21 @@
     }
   },
   "dependencies": {
-    "@voiceflow/alexa-types": "1.52.0",
-    "@voiceflow/api-sdk": "1.33.0",
+    "@voiceflow/alexa-types": "1.52.17",
+    "@voiceflow/api-sdk": "1.36.0",
     "@voiceflow/backend-utils": "2.2.1",
+    "@voiceflow/base-types": "1.0.2",
     "@voiceflow/body-parser": "^1.21.0",
     "@voiceflow/common": "6.5.0",
-    "@voiceflow/general-runtime": "^1.53.5",
-    "@voiceflow/general-types": "1.51.0",
+    "@voiceflow/general-runtime": "1.56.1",
     "@voiceflow/logger": "1.5.7",
     "@voiceflow/verror": "^1.1.0",
+    "@voiceflow/voice-types": "1.1.1",
     "ask-sdk": "^2.9.0",
     "ask-sdk-express-adapter": "^2.11.0",
     "ask-sdk-model": "^1.31.0",
     "aws-sdk": "^2.586.0",
-    "axios": "^0.19.0",
+    "axios": "0.21.1",
```

Comment:
yes, since old(0.19.0) and new(0.21.1) versions have incompatible error types. `0.21.1` is used in the `general-service` so we should sync them to avoid TS errors

Reply:
got it, but couldn't this still be `^0.21.1` to widen the range of allowed axios versions?

Reply:
have no idea, I've installed it via `yarn add axios@0.21.1` - probably yarn-related stuff